db/dev_seeds.rb

```diff
@@ -481,26 +481,26 @@ def unique_document_number
 end
 
 section "Creating Budgets" do
-  Budget.create(
+  budget = Budget.create(
     name: "Budget #{Date.current.year - 1}",
     currency_symbol: "€",
     phase: 'finished'
   )
+
   Budget.create(
     name: "Budget #{Date.current.year}",
     currency_symbol: "€",
     phase: 'accepting'
   )
 
-    (1..([1, 2, 3].sample)).each do |i|
-      group = budget.groups.create!(name: "#{Faker::StarWars.planet} #{i}")
+  (1..([1, 2, 3].sample)).each do |i|
+    group = budget.groups.create!(name: "#{Faker::StarWars.planet} #{i}")
```

Comment:
We need groups for both Budgets, not just for the finished one only

Reply:
Would something like [this](https://gist.github.com/aitbw/f092ecfd990827ce77f8516a29e02ab1) work?

Reply:
I just realized on consul we've a refactor that creates more "descriptive" geozones

Reply:
Hmm 🤔 how should we proceed, then?

Reply:
No problem just that solution will do nice for madrid right now, fixing dev_seeds with "realistic" or "descriptive" content is going to be a long task full of small improvements... best way would be to just on every new feature, increase dev_seeds, as well as everytime we revisit something.. clean related dev_seeds if possible... But for now just that solution 🆗  and we can continue with everything else at madrid :) 👏

Reply:
Great, @bertocq! Amending right away :)